core/frontend/src/ViewState.ts

```diff
@@ -350,6 +350,11 @@ export abstract class ViewState extends ElementState {
    * @see [Views]($docs/learning/frontend/Views.md)
    */
   public async load(): Promise<void> {
+    // If the iModel associated with the viewState is a blankConnection,
+    // then no data can be retrieved from the backend.
+    if (this.iModel.isBlank)
+      return;
+
     const hydrateRequest: HydrateViewStateRequestProps = {};
     this.preload(hydrateRequest);
     const promises = [
```

Comment:
With this fix, displayStyle.load will never be called. Is it what is expected? I mean, where is the saved view state will actually be restored for blank imodel connection?

Reply:
DisplayStyle.load, like ViewState.load, can't do anything useful in the absence of a backend. Recalling a saved view restores not only the view itself but also its display style, model/category selectors, etc. ViewState.load is invoked *after* this state has been restored.